signup.php

```diff
@@ -1,9 +1,21 @@
 <html lang="en">
+
 <head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
 	<title>Sign Up Page</title>
+	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
+	<link rel="stylesheet" type="text/css" href="css/main.css">
 </head>
+
 <body>
-Please sign up
-	<?php include 'forms/user.form.php';?>
+	<div class="wrapper">
+		<img src="https://static01.nyt.com/images/2016/02/09/dining/09COOKING_CHICKENWINGS2/09COOKING_CHICKENWINGS2-superJumbo.jpg" id=bg alt="wings">
+	</div>
```

Comment:
@margeschrec I _believe_ this is the issue here. This should either be css or in the form file. You can confirm by deleting it and testing.